diff -Naur occt7.orig/src/TKIVtk/EXTERNLIB occt7/src/TKIVtk/EXTERNLIB
--- occt7.orig/src/TKIVtk/EXTERNLIB	2017-03-13 05:40:17.000000000 -0300
+++ occt7/src/TKIVtk/EXTERNLIB	2017-08-17 23:43:17.092247530 -0300
@@ -14,6 +14,5 @@
 vtkRenderingCore
 vtkRenderingOpenGL
 vtkRenderingFreeType
-vtkRenderingFreeTypeOpenGL
 vtkFiltersGeneral
 vtkInteractionStyle
diff -Naur occt7.orig/src/TKIVtkDraw/EXTERNLIB occt7/src/TKIVtkDraw/EXTERNLIB
--- occt7.orig/src/TKIVtkDraw/EXTERNLIB	2017-03-13 05:40:17.000000000 -0300
+++ occt7/src/TKIVtkDraw/EXTERNLIB	2017-08-17 23:43:31.276246971 -0300
@@ -20,4 +20,3 @@
 vtkImagingCore
 vtkInteractionStyle
 vtkRenderingOpenGL
-vtkRenderingFreeTypeOpenGL
